cli/src/commands/project.js

```diff
@@ -12,10 +12,12 @@ ${bold('SYNOPSIS')}
 
 ${bold('COMMANDS')}
         info            Display available infos of the current project
+        deploy          Show instructions to deploy your configurations on a server
```

Comment:
Not fan of the command wording, It don't deploy anything, it just show instructions

Reply:
Me neither. But have you a better idea?

Reply:
Update : This command is temporary, until I implement #130 that is aimed to the 1.1 too.
So, I suggest we merge it and I'll update this one before the release